libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/reststop/RestStop.kt

```diff
@@ -11,6 +11,7 @@ import com.mapbox.navigation.base.trip.model.roadobject.location.RoadObjectLocat
  * @param restStopType information about a rest stop. See [RestStopType].
  * @param name name of the rest stop.
  * @param amenities facilities associated with the rest stop. See [AmenityType]
+ * @param guideMapUri URI to guide map image. To use it you must add an access token
```

Comment:
where this token can be taken? do you know @LukasPaczos @mskurydin ?

Reply:
Access token needs to be appended to the url when making the route request